sql/select_test.go

```diff
@@ -69,6 +69,20 @@ func TestMakeConstraints(t *testing.T) {
 		{`a IN (1,2,3) AND b = 1`, []string{"a", "b"}, `[a IN (1, 2, 3), b = 1]`},
 		{`a = 1 AND b IN (1,2,3)`, []string{"a", "b"}, `[a = 1, b IN (1, 2, 3)]`},
 
+		// Prefer IN over >, <, >= and <=.
+		{`a > 1 AND a IN (1, 2)`, []string{"a"}, `[a IN (1, 2)]`},
```

Comment:
something's not right with this one. i guess these tests don't simplify the expression first but perhaps they should? this is really `[a = 2]`